app/routes/projects/index.tsx

```diff
@@ -130,9 +154,11 @@ export default function Projects() {
       missingFacets,
       count,
     },
+    projectMembership,
     ongoingStatuses,
     ideaStatuses,
   } = useLoaderData() as LoaderData;
+  const [showJoinModal, setShowJoinModal] = useState<boolean>(false);
```

Comment:
I don't think we need this state variable, I saw that we can't close the modal by pressing the "X" icon, so I think we can just remove this variable and leave an empty function in the handleClose for the modal